debian/rules

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-DEB_PYTHON_SYSTEM := pysupport
-
-include /usr/share/cdbs/1/rules/debhelper.mk
-include /usr/share/cdbs/1/class/python-distutils.mk
+#include /usr/share/cdbs/1/rules/debhelper.mk
+#include /usr/share/cdbs/1/class/python-distutils.mk
```

Comment:
Why did you comment it out?

Reply:
With out this I see one ends up with the following failure:
```
dh debian/_cdbs_buildflags.mk --with python3 --buildsystem=pybuild
dh: Unknown sequence debian/_cdbs_buildflags.mk (choose from: binary binary-arch binary-indep build build-arch build-indep clean install install-arch install-indep)
test -x debian/rules
dh reverse-config --with python3 --buildsystem=pybuild
dh: Unknown sequence reverse-config (choose from: binary binary-arch binary-indep build build-arch build-indep clean install install-arch install-indep)
```

I have not done much at all with Debian packaging so I do not quite appreciate this significance but I am failing to find a way to build this using both `python-support` and that "era" of rules file and `dh_python` and that style? (However I am similarly broken needing to build on Ubuntu 16.04+.)

Is it required that Diamond still build on such old versions of Ubuntu or can 16.04 become the new lowest version of support?